docs/content/build-and-deploy.md

```diff
@@ -60,10 +60,11 @@ Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/i
 
 ### Cloud deployment
 To start a cloud deployment, follow these steps: 
-1. Upload your server-worker and client-worker assemblies. To do this, in a terminal window, from any directory, run `spatial cloud upload <assembly-name>` from the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). 
+1. Specify the SpatialOS project name you want to deploy to. To do this, locate the spatialos.json file in the root folder of your SpatialOS project and edit the file so the name field matches your SpatialOS project name. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`. (To set up your project in SpatialOS, follow the instructions in the [Setup guide](../setup-and-installing.md).)
```

Comment:
"1. Specify the SpatialOS project name you want to deploy to. To do this, locate the spatialos.json file in the root folder of your SpatialOS project and edit the file so the name field matches your SpatialOS project name. You can find your project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`. (To set up your project in SpatialOS, follow the instructions in the [Setup guide](../setup-and-installing.md).)"

-->
So this is hard...
We have a SpatialOS project in the cloud.
We have a SpatialOS project which is your Unity game.
We need to tie them together, right?
Trying to unravel this....

"1. Tie your Unity-made game project with a SpatialOS project in the cloud. 
<br/>You are allocated an empty SpatialOS project in the cloud when you sign up to SpatialOS; you use this to deploy your Unity-made game project but to do this you need to specify the SpatialOS project name you want to deploy to.  
<br/> To do this, locate the spatialos.json file in the root folder of your Unity-made game project and edit the file so the name field matches your SpatialOS project name. You can find your allocated SpatialOS project name in the [SpatialOS Console](https://console.improbable.io/). It’ll be something like `beta_someword_anotherword_000`. "